app/views/data_entry/index.html.erb

```diff
@@ -0,0 +1,195 @@
+<h1 class="text-center mb-5">Enter your server data</h1>
+<p class="text-center">Is your server connected to the internet?</p>
+<div class="btn-group mb-5" role="group" aria-label="internet-toggle">
+  <input type="radio"
+         class="btn-check"
+         name="btnradio"
+         id="radio-yes"
+         autocomplete="off"
+         <%= 'checked' unless params[:offline_instructions] %>
+  >
+  <label class="btn btn-outline-primary button glaring" for="radio-yes">Yes</label>
+  <input type="radio"
+         class="btn-check"
+         name="btnradio" id="radio-no"
+         autocomplete="off"
+         <%= 'checked' if params[:offline_instructions] %>
+  >
+  <label class="btn btn-outline-primary button glaring" for="radio-no">No</label>
+</div>
+<% unless user_signed_in? %>
```

Comment:
Now only showing the 'Sign in' step card if no user is signed in. This fixed a bug where it would redirect to the home page if a user was signed in, but it also makes sense from a ux perspective